src/js/active-search.js

```diff
@@ -0,0 +1,12 @@
+import $ from "jquery";
+
+
+const initSearch = () => {
+  $('.js-search-btn').click(function (event) {
+      let x = $('.js-search-btn').parents('.box-bands').addClass('box-bands--active');
```

Comment:
The `$('.js-search-btn')` selector used twice. Consider using constant for that.

`const $searchBtn = $('.js-search-btn');`